front/stat.graph.php

```diff
@@ -81,38 +81,38 @@
    case "technicien" :
       $val1    = $_GET["id"];
       $val2    = "";
-      $values  = Stat::getItems($_GET["itemtype"], $_GET["date1"], $_GET["date2"], $_GET["type"]);
```

Comment:
In `stat.class.php'; a link is defind with aprameters as GET; do not know when this is used.

Reply:
nvermind, it' ok